src/Impostor.Api/GlobalUsings.cs

```diff
@@ -0,0 +1 @@
+global using Impostor.Hazel.Abstractions;
```

Comment:
point of this is to prevent everyone needing to change their code? does this extend to api users?

Reply:
The point is to not generate another 3k line changes. It doesn't extend to api users as there is no good way export global usings (except nuget .props hackery)

Reply:
Not extending to users is absolutely fine. You're already generating the 3k removals by removing it from everything

But now that I think about it, importing a class in every single class of a program is kind of bad as well, so this change can stay